Cargo.toml

```diff
@@ -5,7 +5,7 @@ resolver = "2"
 [workspace.package]
 version = "0.9.1"
 edition = "2021"
-rust-version = "1.76"
+rust-version = "1.81"
```

Comment:
wonder if we want to just bump to rust 1.82? (requires bumping everything else to it too)